src/UI/Art/FillRectUnitArtist.java

```diff
@@ -0,0 +1,63 @@
+package UI.Art;
+
+import java.awt.Color;
+import java.awt.Graphics;
+
+import Engine.GameInstance;
+import Terrain.GameMap;
+import Terrain.Location;
+import UI.MapView;
+import Units.Unit;
+
+public class FillRectUnitArtist implements UnitArtist
+{
+	private int tileSizePx = MapView.tileSizePx;
+	
+	GameInstance myGame;
+	GameMap gameMap;
+	
+	public static final Color COLOR_TIRED = new Color(128,128,128,160);
+	
+	public FillRectUnitArtist(GameInstance game)
+	{
+		myGame = game;
+		gameMap = myGame.gameMap;
+	}
+	
+	public void drawUnits(Graphics g)
+	{
+		// TODO: It would be pretty swell if we could just ask the map for a list of units.
```

Comment:
Probs... but then you would also get units in transports, not just those on the map. Still a good idea though, since that should be easy enough to check.